Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM docker.io/library/ubuntu:20.04
+FROM docker.io/library/ubuntu:22.04
```

Comment:
```suggestion
FROM docker.io/library/ubuntu:20.04
```
20.04 should be used when binary is built from suggested `paritytech/ci-unified:bullseye-1.74.0-2023-11-01-v20231204`

Reply:
Reverted back to 20.04